lib/generators/devise/templates/simple_form_for/registrations/edit.html.erb

```diff
@@ -30,6 +30,6 @@
 
 <h3><%= t(".cancel_my_account") %></h3>
 
-<p><%= t(".unhappy") %> <%= link_to t(".cancel_my_account"), registration_path(resource_name), data: { confirm: t(".are_you_sure") }, method: :delete %></p>
+<div><%= t(".unhappy") %> <%= link_to t(".cancel_my_account"), registration_path(resource_name), data: { confirm: t(".are_you_sure"), turbo_confirm: t('.are_you_sure') }, method: :delete %></div>
```

Comment:
I recommend to use `button_to` instead of `link_to` according to [the current code](https://github.com/heartcombo/devise/blob/v4.9.0/lib/generators/templates/simple_form_for/registrations/edit.html.erb#L33).

```suggestion
<div><%= t(".unhappy") %> <%= button_to t(".cancel_my_account"), registration_path(resource_name), data: { confirm: t(".are_you_sure"), turbo_confirm: t(".are_you_sure") }, method: :delete %></div>
```